awswrangler/catalog/_create.py

```diff
@@ -323,6 +323,85 @@ def _create_parquet_table(
     )
 
 
+def _create_legacy_delta_table(
```

Comment:
It seems that this private method is identical to the `_create_parquet_table` method. Given that it's private anyways, perhaps we can just reuse the `_create_parquet_table` instead. If there is a compelling reason to have a dedicated method in the future, it can easily be done without impact to the user (since it's private)